### DISCUSSION TITLE
how i can assign ip address to alpine client eth1 interface?

### DISCUSSION THREAD

**Discussion by nanisyako** [srl-labs/containerlab#1192] [Q&A] [answered]
it seems the alpine client doesn't have /etc/network/interfaces by default and since eth0 is on dhcp how I can statically configure eth1 for ghcr.io/hellt/network-multitool linux container?

**Comment by hellt** [comment, marked as answer]
Hi @nanisyako 
you have to use [exec](https://containerlab.dev/manual/nodes/#exec) and call ip/ifconfig utilities as with regular containers

Examples:

* https://github.com/srl-labs/srl-telemetry-lab/blob/7adcfed24bb7450f0e9a88ab72b553ea3d018104/st.clab.yml#L66
* https://github.com/srl-labs/containerlab/blob/ba3475c6c4404aff030e246fe702ec58a6c1683d/lab-examples/ixiac01/ixiac01.clab.yml#L9
